### ISSUE TITLE
V1.01 all good

### ISSUE THREAD

**Issue by danielforster** [njfaria/dalylog#11]
Sorry about not getting back sooner, my kit was in transit and now on the other side of the world with V1.01 working fine with MQTT temp and all existing readings). Cheers for all your hard work as always.. Daniel

**Comment by SunnyPerth** [comment]
Yup - Can confirm V1.01 has been running for a bit now - all good!

**Comment by kxn25** [comment]
is ther any updates planed?
can the error status be added?
Or is there a better solution to monitor Daly BMS?
